src/common/providers/https.ts

```diff
@@ -169,6 +169,56 @@ export interface CallableRequest<T = any> {
   rawRequest: Request;
 }
 
+/** How a task should be retried in the event of a non-2xx return. */
+export interface TaskRetryConfig {
+  // If left unspecified, will default to 5
```

Comment:
Q: Should be leave this comment out? Wondering if this comment is better left out since it's the responsibility of the CLI/Cloud Task Queue to set the defaults if missing.

Reply:
I think the bug is that these aren't customer docs.